content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md

```diff
@@ -283,23 +350,38 @@ the Pods' relative priority to determine the eviction order, because inodes and
 requests.
 
 The kubelet sorts pods differently based on whether the node has a dedicated
-`imagefs` filesystem:
+`imagefs` or `containerfs` filesystem:
 
-#### With `imagefs`
+#### With `imagefs` (`nodefs` and `imagefs` filesystems are separate) {#with-imagefs}
 
-If `nodefs` is triggering evictions, the kubelet sorts pods based on `nodefs`
-usage (`local volumes + logs of all containers`).
+- If `nodefs` triggers evictions, the kubelet sorts pods based on `nodefs`
+  usage (`local volumes + logs of all containers`).
 
-If `imagefs` is triggering evictions, the kubelet sorts pods based on the
-writable layer usage of all containers.
+- If `imagefs` triggers evictions, the kubelet sorts pods based on the
+  writable layer usage of all containers.
 
-#### Without `imagefs`
+#### Without `imagefs` (`nodefs` and `imagefs` use the same filesystem) {#without-imagefs}
+
+- If `nodefs` triggers evictions, the kubelet sorts pods based on their
+  total disk usage (`local volumes + logs and a writable layer of all containers`).
+
+#### With `containerfs` (`imagefs` and `containerfs` have been split) {#with-containersfs}
```

Comment:
Note: the order of these is odd. Since `with imagefs`  was previously first, it implies to me it was ordered by most customized to least, so the `with containerfs` section should be above it. I would really prefer it go `without imagefs or containerfs` `with imagefs` `with and imagefs containerfs`

Reply:
Sounds good.

Reply:
Done.